src/appkit.cs

```diff
@@ -20955,7 +20989,23 @@ partial interface NSWindow : NSAnimatablePropertyContainer, NSUserInterfaceItemI
 		bool DocumentEdited  { [Bind ("isDocumentEdited")] get; set; }
 	
 		[Export ("isVisible")]
-		bool IsVisible  { get; set; }
+		bool IsVisible  {
+			get;
+#if !XAMCORE_5_0
+			// https://github.com/xamarin/xamarin-macios/issues/14359
+			[Obsolete ("Setting 'IsVisible' will probably behave unexpectedly, since it comes from the NSScripting protocol (and not like the getter, which is defined on the NSWindow type). If this is the expected behavior, call 'SetIsVisible(bool)' instead.")]
```

Comment:
Maybe point to orderFront/orderOut?

Reply:
@Therzok do you have a code snippet or the exact code to use?

Reply:
`window.OrderOut(null)` for IsVisible=false, `window.OrderFront(null)` to make the window visible, on top of others.

See https://developer.apple.com/documentation/appkit/nswindow/1419672-orderwindow?language=objc for a more general version.

Reply:
Ok, I've improved the message a bit, pointing to OrderFront/OrderOut.

Reply:
Thank you!